.github/renovate.json5

```diff
@@ -0,0 +1,3 @@
+{
+  extends: ['github>seek-oss/rynovate'],
```

Comment:
Probably should've checked this, but does rynovate pick up github action updates?

Reply:
We should be able to tell when it makes the dashboard, if it's under the detected dependency updates or not. I don't remember off the top of my head.